### DISCUSSION TITLE
part-ii/2015-95/

### DISCUSSION THREAD

**Discussion by tripos-comments** [tripos-education/maths-tripos-questions#285] [Question comments]
Part II 2015 Number Theory: Paper 3, Section II, H 

https://questions.tripos.org/part-ii/2015-95/

**Comment by walrus1729** [comment]
<details>
<summary>Solution</summary>

Everything except the last part is bookwork.

For the last part, we compute the continued fraction expansion of $\sqrt{53}$. We get that $a_0 = 7$, $a_1 = 3$, $a_2 = 1$, $a_3 = 1$, so we get the convergents

$$
(p_0,q_0) = (7,1) \\
(p_1,q_1) = (22,3) \\
(p_2,q_2) = (29,4) \\
(p_3,q_3) = (51,7) \\
$$

We find that $29^2 - 53 \cdot 4^2 = -7$ and $51^2 - 53 \cdot 7^2 = 4$.

</details>
